src/app/pages/network/network.component.ts

```diff
@@ -664,4 +657,22 @@ export class NetworkComponent implements OnInit, OnDestroy {
     }
     return true;
   }
+
+  private openInterfaceForEditFromRoute(): void {
+    const state = this.navigation.extras.state as { editInterface: string };
+    if (!state?.editInterface) {
+      return;
+    }
+
+    this.ws.call('interface.query', [[['id', '=', state.editInterface]]])
+      .pipe(untilDestroyed(this))
+      .subscribe((interfaces) => {
+        if (!interfaces[0]) {
+          return;
+        }
+
+        const form = this.slideInService.open(InterfaceFormComponent);
+        form.setInterfaceForEdit(interfaces[0]);
```

Comment:
There should be some loading indicator mechanism here. As once you navigate to the network page, you're sitting there waiting for something to happen (i.e., the form to open) with no indication that UI is working on it. You could assume it was a miss-click or UI failed to understand the action required if the request takes long enough.